config/default.yaml

```diff
@@ -18,7 +27,7 @@ influxdb:
   password: 'root'
   use_udp: false
   retentionPolicy: 'default'
-  # Acceptable version are: '0.8' and '0.9'
+  # All version other than '0.9' will default to '0.8'
```

Comment:
This is for backwards compatibility. If a user updates their bucky server and copies over their old default.yaml file it will work as it did before the version option was added.